Dynamic Programming/best_time_to_buy_and_sell_stock.py

```diff
@@ -0,0 +1,11 @@
+def maxProfit(prices):
```

Comment:
Could you add description and the way you approach the problem with comments ? Thanks!